imports/plugins/included/shipping-shippo/server/hooks/rates.js

```diff
@@ -4,24 +4,28 @@ import { Logger, Reaction, Hooks } from "/server/api";
 
 // callback ran on getShippingRates hook
 function getShippingRates(previousQueryResults, cart) {
+  const marketplaceSettings = Reaction.getMarketplaceSettings();
+  const { public: merchantShippingRates } = marketplaceSettings;
```

Comment:
Your suggestion doesn't work because if marketplaceSettings is undefined it throws an error and we are done. That's the problem the destructuring is trying to solve (and I think it does but I will test again)

Reply:
Good point. Possible I'm mistaken about the way the destructuring assignment works in this case, but based on the MDN docs and the jsbin tests I ran, I think it's assigning the whole public object to `merchantShippingRates`

Reply:
I've updated the JSBin to show how it actually does work

Reply:
Ah wait, I see what you are saying

Reply:
What decision did you make here?